src/routes/postProducts.js

```diff
@@ -0,0 +1,5 @@
+import Storage from '../services/storage';
+
+export default function getProducts(req, res) {
+    res.send(Storage.add('products', req.body));
```

Comment:
Hmm, doesn't work for me for either x-www-form-urlencoded or application/json request.

Reply:
Must mork, please check format for application/json: https://i.imgur.com/RurCQIP.png